packages/storybook-css/src/NavBar.stories.tsx

```diff
@@ -123,4 +123,50 @@ export const WrapText: Story = {
   name: 'Wrap long text in items',
 };
 
+export const WithCurrentItem: Story = {
+  args: {
+    children: (
+      <NavList>
+        <NavListLink href="#">Wonen en leven</NavListLink>
+        <NavListLink href="#" current="page">
+          Werk en inkomen
+        </NavListLink>
+        <NavListLink href="#">Ondernemen</NavListLink>
+        <NavListLink href="#">Bestuur en organisatie</NavListLink>
+      </NavList>
+    ),
+  },
+  name: 'Navigation list with current item',
+};
+
+const alignStyles: React.CSSProperties = {
```

Comment:
De conventie is om `import type { CSSProperties }` te doen ipv `React.CSSProperties`